### PR TITLE
docs(currentRefinedValues): documents the widget

### LINKED ISSUE #781: documentation: add clearRefinements documentation
clearRefinements is already implemented and used, now it's time to add it into our beautiful website

cc @Jerskouille

### PR DESCRIPTION
**Pull request by bobylito**
Fix #781
Well the issue was not clear, but as with @Jerskouille, clearAll was already documented and currentRefinedValues was not so here it is :)

### CODE REVIEW

File: docs/js/doc.js [hunk 1]
```diff
@@ -16,9 +16,16 @@
...
+      hitsPerPage: 6,
+      disjunctiveFacets: ['brand'],
+      disjunctiveFacetsRefinements: {
+        brand:['Samsung'] 
+      }
+    };
     var codeSnippets = q('.code-sample-snippet:not(.start):not(.config):not(.ignore)');
     var configSnippet = "var search = instantsearch({appId: '$appId', apiKey: '$apiKey'," +
-      "indexName: '$indexName', searchParameters: {hitsPerPage: 6}});\n"
+      "indexName: '$indexName', searchParameters: " + JSON.stringify(initialSearchState) + "});\n"
```
Comment (vvo) [inline docs/js/doc.js]: Are we sure this is what we want to show to the user? I thought we just wanted to show the hitsPerPage

Comment (bobylito) [inline docs/js/doc.js]: This is the initial configuration for all the widgets, it's not shown to the user. It's required for this feature, as it doesn't display anything if nothing is refined :)

File: docs/documentation.md [hunk 2]
```diff
@@ -912,6 +912,40 @@ instantsearch.widgets.clearAll(options);
 
 <div id="clear-all" class="widget-container"></div>
 
+#### currentRefinedValues
+
+<div class="codebox-combo">
+
+<img class="widget-icon pull-left" src="../img/icon-widget-clearall.svg">
+This widget list all the refinements currently applied. It also lets
```
Comment (Jerskouille) [inline docs/documentation.md]: "lists" , no?

**Comment by Jerskouille** [comment]
:+1:

File: docs/documentation.md (hunk 2 above)
Comment (bobylito) [inline docs/documentation.md]: good catch!

**Pull request merged**